a3p-integration/proposals/z:acceptance/tsconfig.json

```diff
@@ -10,6 +10,6 @@
     "strictNullChecks": true,
     "noImplicitThis": true,
     // XXX synthetic-chain has some errors
-    "skipLibCheck": true,
+    "skipLibCheck": true
```

Comment:
unrelated and unnecessary

Reply:
Oh it was an auto fix from `yarn format`, however resolved in https://github.com/Agoric/agoric-sdk/commit/712f97060737ce2a54d4e27ba2047142da6585ab now